lib/auth/auth.go

```diff
@@ -2433,7 +2437,7 @@ func (a *Server) CreateAccessRequest(ctx context.Context, req types.AccessReques
 			Expires: req.GetAccessExpiry(),
 		},
 		Roles:                req.GetRoles(),
-		RequestedResourceIDs: req.GetRequestedResourceIDs(),
+		RequestedResourceIDs: requestedResourcesStr,
```

Comment:
As in `.proto` file: I believe we would be better off with strongly typed fields here.